Plugins/NativeInput/Source/Unix/NativeInput.cpp

```diff
@@ -0,0 +1,10 @@
+#include "../Shared/NativeInput.h"
+
+namespace Babylon::Plugins
+{
+    bool NativeInput::Impl::HasMouse()
+    {
+        // TODO: Currently assuming a mouse is present.  Need to evaluate possibly detecting this programmatically.
```

Comment:
I would recommend filing an issue to address this later.

Reply:
Added as task to https://github.com/BabylonJS/BabylonNative/issues/956